iron-fit-behavior.html

```diff
@@ -503,24 +506,24 @@
       return target.getBoundingClientRect();
     },
 
-    __isAlignRight: function(hAlign, rect, positionRect, fitRect) {
+    __isAlignRight: function(hAlign, size, positionRect, fitRect) {
       if (hAlign === 'right') {
         return true;
       }
       if (hAlign === 'auto') {
         return positionRect.left + positionRect.width/2 > fitRect.left + fitRect.width/2 ||
-          (!this.noOverlap && positionRect.left < 0 && positionRect.right - rect.width > positionRect.left);
+          (!this.noOverlap && positionRect.left < 0 && positionRect.right - size > positionRect.left);
```

Comment:
I recommend splitting this `return` up so that the different significant values being compared are represented by temp variables, for readability 😵